swift/common/middleware/proxy_logging.py

```diff
@@ -138,6 +182,17 @@ def __init__(self, app, conf, logger=None):
             conf.get('reveal_sensitive_prefix', 16))
         self.check_log_msg_template_validity()
 
+        self.perfdata = config_true_value(conf.get('perfdata', 'false'))
+        self.perfdata_user_agents = None
+        if self.perfdata:
+            pattern_dict = {k: v for k, v in conf.items()
+                            if k.startswith('perfdata_user_agent')}
+            self.perfdata_user_agents = [re.compile(pattern_dict[k])
+                                         for k in sorted(pattern_dict.keys())]
+            if not self.perfdata_user_agents:
+                self.logger.warn('No user_agent pattern defined, '
+                                 'all clients will add perfdata.')
```

Comment:
Clients do not add perfdata. I suggest "performance data will be logged for every request".